SmartSaver.MVC/Views/Authentication/Login.cshtml

```diff
@@ -9,29 +9,57 @@
 </head>
 
 <br />
-<h1>User Login</h1>
 <br />
```

Comment:
I see that you use `<br>` everywhere in your code. Don't know which one is better, but use `<br>` in here as well